training/src/anemoi/training/config/dataloader/native_grid.yaml

```diff
@@ -50,27 +50,33 @@ grid_indices:
 # See https://anemoi-datasets.readthedocs.io
 # ============
 
-dataset: ${hardware.paths.data}/${hardware.files.dataset}
+dataset_defaults:
+
+  dataset: 'aifs-ea-an-oper-0001-mars-o96-1979-2022-6h-v6'
+  # Either a path to a dataset, a dataset name or a nested dictionary
+  # when using dataset names, make sure the search path is set in the anemoi-datasets config
+  # as described in the anemoi-datasets documentation
+  # https://anemoi.readthedocs.io/projects/datasets/en/latest/datasets/using/configuration.html
+
+  frequency: ${data.frequency}
+  drop: []
+  # start: ...
+  # end: ...
+  # select: []
 
 training:
-  dataset: ${dataloader.dataset}
+  dataset: ${dataloader.dataset_defaults}
   start: null
   end: 2020
```

Comment:
What is the relation of `start`, `end` , etc. specified here to the values specified in `dataloader.dataset_defaults`?

Do settings here (like `end=2020`) selectively overwrite settings made in the respective `dataset`?